src/test/java/com/ss/greencity/util/WaitsSwitcher.java

```diff
@@ -0,0 +1,91 @@
+package com.ss.greencity.util;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+/**
+ * A class intended to help easily switching between implicit and explicit waits
+ */
+public final class WaitsSwitcher {
+    private WebDriver driver;
+    private long defaultImplicitWait;
+    private long defaultExplicitWait;
+
+    public WaitsSwitcher(WebDriver driver, long defaultImplicitWait, long defaultExplicitWait) {
+        this.driver = driver;
+        this.defaultImplicitWait = defaultImplicitWait;
+        this.defaultExplicitWait = defaultExplicitWait;
+    }
+    public WaitsSwitcher(WebDriver driver, long defaultWait) {
+        this.driver = driver;
+        defaultImplicitWait = defaultWait;
+        defaultExplicitWait = defaultWait;
+    }
+    public WaitsSwitcher(WebDriver driver) {
+        this.driver = driver;
+        defaultImplicitWait = 1;
+        defaultExplicitWait = 5;
+    }
+
+    /**
+     * A wrap for Thread.sleep
+     * @param seconds - time in seconds (could be decimal)
+     */
+    public static void sleep(double seconds) {
```

Comment:
Use this wrap (obgortka) instead of Thread.sleap(), because you will have no need to write everywhere try-catch.
But try better to make use of explicit waits (see below)